main.go

```diff
@@ -195,7 +212,13 @@ func setupIndexes(ctx context.Context, mgr ctrl.Manager, cfg *configapi.Configur
 	return err
 }
 
-func setupControllers(mgr ctrl.Manager, cCache *cache.Cache, queues *queue.Manager, certsReady chan struct{}, cfg *configapi.Configuration, serverVersionFetcher *kubeversion.ServerVersionFetcher) {
+func setupControllers(
```

Comment:
nit: please revert if this is just a formatting change of otherwise untouched code, this will make review easier.

Reply:
reverted